python/desc/slrealizer/SLRealizer.py

```diff
@@ -1,27 +1,45 @@
-import plotting
+import desc.slrealizer
 import pandas as pd
 import matplotlib.pyplot as plt
 import pylab
 import matplotlib
 import math
 
 class SLRealizer(object):
-    
+
     scale_factor = 2
-    
+
     # Maybe make a separate method for it
     # Best OOP practice?
 
-    def __init__(self, lens = None, observation = "../../data/twinkles_observation_history.csv"):
+    def __init__(self, catalog=None, observation="../../data/twinkles_observation_history.csv"):
```

Comment:
'lens' was a confusing and innaccurate name for this object, which is an `om10.DB` object or equivalent, ie a _catalog_.